### PR TITLE
Route doc updates and fixes

### PR DESCRIPTION
**Pull request by doublestranded**
- [x] Accessibility values for routes now reflect API
- [x] shapes_txt_with_dist_traveled require shape_dist_traveled in stops and stop_times
- [x] Explanation for new distance algorithm

### CODE REVIEW

**Comment by rmglennon** [comment]
Preview here: https://precog.mapzen.com/transitland/www-transit-land/route-doc-updates-and-fixes/documentation/datastore/routes-and-route-stop-patterns.html

**Comment by rmglennon** [comment]
Hey @doublestranded, thanks for the docs!

Few comments:
- Maybe reword the new paragraph to avoid "we" (yes, yes, it makes it passive voice in cases...), and the "we" in another existing paragraph while we're at it. 😄 
- I don't quite understand "segment matching possibilities". Maybe we could have a chat about the new paragraph?

**Comment by doublestranded** [comment]
Thanks @rmglennon! I've made your suggested changes, did some re-writes and added an image to explain a topic.

**Comment by rmglennon** [comment]
:shipit:
